### ISSUE TITLE
Memory issues

### ISSUE THREAD

**Issue by krystofurr** [disaster37/rancher-alfresco#7]
Hello,

I've been working with your docker build for Alfresco and was wondering if its normal to have memory issues when manually building from the Dockerfile instead of pulling from the docker hub.  I never seem to have issues pulling the image down from the hub and running the container, but when downloading the github repository and building from the Dockerfile, I get the following error during the build:

`0.97 GB of RAM is not enough to run Alfresco for test or production purposes. Alfresco might not work correctly and if it does, will not perform well. Upgrade the server to have at least 2.0 GB of RAM, and optimally 4.0 GB of RAM.
The command '/bin/sh -c /app/assets/install_alfresco.sh' returned a non-zero code: 1`

I'm running docker toolbox in a Windows 10 environment.  I noticed that my virtual box VM only have 1024MB of RAM set for it which is likely causing the issue, but why does it work fine when I pull the image down from docker hub?

**Comment by disaster37** [comment]
Hi,

like you say, your VM have no enough memory. In fact, we use the official install script that check the machine requirement before start installation. And the script check that you are the minimum memory for Alfresco before start installation. In not really good situation for docker context. Maybee there aare king of argument to not check the memory requirement during alfresco installation...

**Comment by krystofurr** [comment]
Ah I see.  So your staying when you download from the docker hub there may be an argument that prevents the memory check ?  How could I disable that check for the Dockerfile build?

**Comment by dm-conti** [comment]
I have the same problem on Mac OS X but in my case it detects 1.95Gb which is always lower than the minimum requirement of 2Gb.

**Comment by disaster37** [comment]
Yep, Alfresco is not packaged to be include on Docker. So we need to use the Alfresco standard installer when build Docker image. The matter is that the Alfresco installer check the server requierement before process the installation. So if you haven't 2go of Ram on the machine that buid the docker image, you get your error. There are no error when the Docker image is build by docker hub or quay.io because of the server that run the build has more than 2go of ram.

I hope it's clear. I don't found any option on Alfresco installer to disable the memory check requirement.
